src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Product.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Core\Model\Product" shortName="Product" processor="Sylius\Bundle\ApiBundle\StateProcessor\ProductProcessor">
+        <validationContext>
+            <values>
+                <value name="groups">
+                    <values>
+                        <value>sylius</value>
+                    </values>
+                </value>
+            </values>
+        </validationContext>
+
+        <order>
+            <values>
+                <value name="code">ASC</value>
+            </values>
+        </order>
+
+        <operations>
+            <operation class="ApiPlatform\Metadata\GetCollection" uriTemplate="/admin/products">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>sylius:admin:product:index</value>
+                            </values>
+                        </value>
+                    </values>
+                </normalizationContext>
+                <filters>
+                    <filter>sylius.api.product_name_filter</filter>
+                    <filter>sylius.api.product_order_filter</filter>
+                    <filter>sylius.api.product_taxon_code_filter</filter>
+                    <filter>Sylius\Bundle\ApiBundle\Filter\Doctrine\ChannelsAwareChannelFilter</filter>
+                    <filter>Sylius\Bundle\ApiBundle\Filter\Doctrine\TranslationOrderNameAndLocaleFilter</filter>
```

Comment:
It would be good to make the names of the services here consistent and not use FQCN